platforms/ios/framework/src/TGGeoPolygon.h

```diff
@@ -24,39 +25,39 @@ TG_EXPORT
 /**
  Inits a polygon and allocates enough memory to hold `size` geographic coordinates in the polygon paths.
  */
-- (instancetype)initWithSize:(unsigned int)size;
+- (instancetype)initWithSize:(NSUInteger)size;
 
 /**
- Starts a new polygon path for a given `TGGeoPoint`.
+ Starts a new polygon path for a given coordinate.
 
- @param latlon the coordinate to start a polygon path
+ @param point the coordinate to start a polygon path
  */
-- (void)startPath:(TGGeoPoint)latlon;
+- (void)startPath:(CLLocationCoordinate2D)point;
 
 /**
- Starts a new polygon path for a given `TGGeoPoint` and allocate enough memory to hold `size` coordinates
+ Starts a new polygon path for a given coordinate and allocate enough memory to hold `size` coordinates
  in the polygon geometry.
 
- @param latlon the coordinate to start a polygon path
+ @param point the coordinate to start a polygon path
  @param size the size of the polygon path
  */
-- (void)startPath:(TGGeoPoint)latlon withSize:(unsigned int)size;
+- (void)startPath:(CLLocationCoordinate2D)point withSize:(NSUInteger)size;
 
 /**
  Adds a single coordinate to the current polygon path.
 
- @param latlon the coordinate to add to the current path
+ @param point the coordinate to add to the current path
 
  @note This operation is a no-op if no polygon path were already started.
  */
-- (void)addPoint:(TGGeoPoint)latlon;
+- (void)addPoint:(CLLocationCoordinate2D)point;
 
 /**
- Returns a pointer to a sequence of `TGGeoPoint` with a total array length equal to `-[TGGeoPolygon count:]`.
+ Returns a pointer to a sequence of coordinates with a total array length equal to `-[TGGeoPolygon count:]`.
 
  @return a pointer to the list of polygons coordinates
  */
-- (TGGeoPoint*)coordinates;
+- (CLLocationCoordinate2D *)coordinates;
 
 /**
  The list of polygon `rings` contained in this geometry.
```

Comment:
This is was a few lines below, is there a reason you did not use system type (`NSInteger`) for the ` - (int *)rings;` below?